front/components/assistant_builder/AssistantBuilderPreviewDrawer.tsx

```diff
@@ -187,6 +191,38 @@ export default function AssistantBuilderPreviewDrawer({
                     }}
                     icon={XMarkIcon}
                   />
+                  <DropdownMenu.Item
```

Comment:
It's just 3 buttons calling each a function passed as attribute of the component, I honestly think it's more convenient to have it inline instead of always having to pomme click.